.github/workflows/build_and_test_clang_debug.yml

```diff
@@ -58,9 +58,7 @@ jobs:
                               -DUSE_FAKE_CLOCK_IN_TIME_SERVICE=TRUE\" "\
               && script -q -e -c "make test"
         - name: Check ERROR/FATAL logs
-          id: err_report
           run: |
-            chmod +x "./.github/success_action_if_err_logs_exist.sh"
```

Comment:
Since, this line is removed, pls make sure `success_action_if_err_logs_exist.sh` has +x access rights and then commit the same.

Reply:
yes Pankaj, I have committed the same already.
ssitu@ubuntu ~/scan_logs/concord-bft/.github (scan-replica-logs) $ ls -lrt success_action_if_err_logs_exist.sh 
-rwxrwxr-x 1 ssitu ssitu 387 Sep 30 01:59 success_action_if_err_logs_exist.sh